packages/manager/apps/pci-instances/src/data/api/instance.ts

```diff
@@ -50,7 +52,7 @@ export const getInstances = (
         searchValue,
       },
     })
-    .then((response) => response.data);
+    .then((response: AxiosResponse<TAggregatedInstanceDto[]>) => response.data);
```

Comment:
the type should be when calling get method :

`v6.get<TAggregatedInstanceDto[]>(...`